src/amqp.rs

```diff
@@ -1,13 +1,19 @@
 use lapin::tcp::{OwnedIdentity, OwnedTLSConfig};
 use vector_config::configurable_component;
 
-/// Connection options for `AMQP`.
+/// AMQP connection options.
 #[configurable_component]
 #[derive(Clone, Debug)]
 pub(crate) struct AmqpConfig {
-    /// URI for the `AMQP` server.
+    /// URI for the AMQP server.
     ///
-    /// Format: amqp://<user>:<password>@<host>:<port>/<vhost>?timeout=<seconds>
+    /// The URI has the format of
+    /// `amqp://<user>:<password>@<host>:<port>/<vhost>?timeout=<seconds>`.
```

Comment:
I guess the unofficial guide I personally follow is that backticks imply some sort of mechanical format i.e. something parsed by a program where the format must be followed as-is.

For example, it makes sense to talk about the "`amqp` source" because `amqp` is the source type that must be specified in the configuration, but "AMQP" is the acronym, as a standalone word: formatting it as a monospaced codeblock doesn't make sense.

Similarly, for the AMQP URI format, we're indicating what the valid URI format is, so we wrap it in backticks to apply the appropriate style to signify that.